### ISSUE TITLE
mat-autocomplete causes invalid aria-owns value

### ISSUE THREAD

**Issue by ukrukarg** [angular/material2#12332]
#### Bug, feature request, or proposal:
Bug, invalid aria-owns value for mat-autocomplete.

#### What is the expected behavior?
aria-owns value should be correct.

#### What is the current behavior?
I got this while running Axe checks on a page using mat-autocomplete.

#### What are the steps to reproduce?
Key | Value 
-- | --
Snippet of HTML of the Element | `<input class="mat-input-element mat-form-field-autofill-control cdk-text-field-autofill-monitored ng-untouched ng-pristine ng-valid" matinput="" autocomplete="off" role="combobox" aria-autocomplete="list" aria-expanded="false" aria-owns="mat-autocomplete-1" id="mat-input-1" aria-invalid="false" aria-required="false">`
Impact | critical
Targets | #mat-input-1
FailureSummary | Fix all of the following: Invalid ARIA attribute value: aria-owns="mat-autocomplete-1"
CheckResult ID | aria-valid-attr-value 
Impact | critical 
Message | Invalid ARIA attribute value: aria-owns="mat-autocomplete-1"
Data | [aria-owns="mat-autocomplete-1"]

I'm a Googler and can help repro when asked.

**Comment by ukrukarg** [comment]
Internal bug: b/111752369
